server/src-lib/Hasura/Server/Logging.hs

```diff
@@ -146,6 +146,7 @@ data OperationLog
   , olResponseSize       :: !(Maybe Int64)
   , olQueryExecutionTime :: !(Maybe Double)
   , olQuery              :: !(Maybe Value)
+  , olRawQuery           :: !(Maybe Value)
   , olError              :: !(Maybe Value)
   } deriving (Show, Eq)
 $(deriveToJSON (aesonDrop 2 snakeCase) ''OperationLog)
```

Comment:
It probably makes sense to enable the [`omitNothingFields`](https://hackage.haskell.org/package/aeson-1.4.5.0/docs/Data-Aeson.html#v:omitNothingFields) option here to suppress generating explicit `null`s in the output, since it isn’t very helpful to get things like `"raw_query": null` in log messages.